common/djangoapps/mailchimp_pipeline/tests/test_signals.py

```diff
@@ -122,15 +125,15 @@ def test_sync_grantee_optin_with_mailchimp(self):
             "PUT", url=expected_url, headers=ANY, data=json.dumps(expected_data), auth=ANY, params=ANY)
 
     @patch("nodebb.signals.handlers.get_current_request", autospec=True)
-    def test_sync_extended_profile_with_mailchimp(self, nodebb_request):
+    def test_sync_extended_profile_with_mailchimp(self, nodebb_request):  # pylint: disable=unused-argument
         """
-            Test if the Extended profile post-save signal is generated and its handler is sending
-            user extended profile data perfectly to the MailChimp expected URL
-            :param nodebb_request: Mocked request used in function that sync user extended profile with NodeBB
+        Test if the Extended profile post-save signal is generated and its handler is sending
+        user extended profile data perfectly to the MailChimp expected URL
+        :param nodebb_request: Mocked request used in function that sync user extended profile with NodeBB
```

Comment:
Lets use the same format for docstring here as well as mentioned above

Reply:
Not under the scope of the ticket. I just removed the extra spacing.